packs/lamp/config.yml

```diff
@@ -0,0 +1,41 @@
+
+# Name of the pack
+name: LAMP stack
+
+# Description of the pack
+description: |+
+  This LAMP observability pack allows you to monitor a LAMP stack (Linux, Apache, MySQL, PHP).
+  It includes a specific dashboard to get observability for an entire LAMP stack.
+
+  NOTE: Observability pack references are currently not yet implemented,
```

Comment:
It's not clear to me what this is trying to say

Reply:
But i guess we are saying each pack has to be installed manually at this point as references aren't integrated.